vortexasdk/__init__.py

```diff
@@ -13,3 +13,6 @@
 
 # noinspection PyUnresolvedReferences
 from vortexasdk.version import __version__
+
+# noinspection PyUnresolvedReferences
+from vortexasdk.check_setup import run_all_checks
```

Comment:
while we're here, i've added this to the init file.

This allows you to write:

```
import vortexasdk as v
v.run_all_checks()
```